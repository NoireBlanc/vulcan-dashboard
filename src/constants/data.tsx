export const SHIPMENT_TYPES = [
    'RMA',
    'Outbound',
    'DC2Store_Inbound',
    'Inbound',
    'WFS_Inbound',
    'Vision_Outbound',
    'Pharmacy_Outbound',
];

export const SENDER_TYPES = ['DC', 'Home', 'Direct', 'Marketplace', 'DSV', 'Store', 'MP'];

export const CHANNELS = ['S2H', 'Delivery', 'S2S', 'Pickup', 'SFS', 'EA'];
