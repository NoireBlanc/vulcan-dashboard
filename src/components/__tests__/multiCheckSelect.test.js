import React from 'react';
import { act } from 'react-dom/test-utils';
import MultiCheckSelect from '../multiCheckSelect';
import { SHIPMENT_TYPES } from '../../constants/data';
import { mount, configure } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

configure({ adapter: new Adapter() });
React.useLayoutEffect = React.useEffect;

describe('<MultiCheckSelect />', () => {
    it('should render', () => {
        act(() => {
            let wrapper = mount(<MultiCheckSelect names={SHIPMENT_TYPES} label={'Test'} />);
            const { container } = wrapper;
            expect(container).toMatchSnapshot();
        });
    });
});
