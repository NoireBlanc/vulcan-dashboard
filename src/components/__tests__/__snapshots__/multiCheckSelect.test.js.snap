// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MultiCheckSelect /> should render 1`] = `undefined`;
